flake.nix

```
@@ -0,0 +1,69 @@
+{
+  description = "Instant, easy, predictable shells and containers";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+
+        lastTag = "0.13.1";
+
+        # Add the commit to the version string, in case someone builds from main
+        getVersion = pkgs.lib.trivial.pipe self [
+          (x: "${lastTag}")
+          (x: if (self ? revCount)
+              then "${x}-${self.shortRev}"
+              else "${x}-${self.dirtyShortRev}")
+        ];
+
+        # Run `devbox run update-flake` to update the vendorHash
+        vendorHash = if builtins.pathExists ./vendor-hash
+                     then builtins.readFile ./vendor-hash
+                     else "";
+
+        buildGoModule = pkgs.buildGo123Module;
+
+      in
+      {
+        inherit self;
+        packages.default = buildGoModule rec {
+          pname = "devbox";
+          version = getVersion;
+
+          src = ./.;
+
+          inherit vendorHash;
+
+          ldflags = [
+            "-s"
+            "-w"
+            "-X go.jetpack.io/devbox/internal/build.Version=${version}"
+          ];
+
+          # Disable tests if they require network access or are integration tests
+          doCheck = false;
+
+          nativeBuildInputs = [ pkgs.installShellFiles ];
+
+          postInstall = ''
+            installShellCompletion --cmd devbox \
+              --bash <($out/bin/devbox completion bash) \
+              --fish <($out/bin/devbox completion fish) \
+              --zsh <($out/bin/devbox completion zsh)
+          '';
+
+          meta = with pkgs.lib; {
+            description = "Instant, easy, predictable shells and containers";
```

Should we match the repo description?

```suggestion
            description = "Instant, easy, and predictable development environments";
```